types/src/jwks/mod.rs

```diff
@@ -268,6 +316,154 @@ impl QuorumCertifiedUpdate {
     }
 }
 
+/// To represent a DELETE operation using existing `ProviderJWKs` types,
+/// we put a `RSA_JWK` and set `n` to be this special value.
+pub const DELETE_COMMAND_INDICATOR: &str = "THIS_IS_A_DELETE_COMMAND";
+
+/// Represents a key-level JWK update.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, CryptoHasher, BCSCryptoHash)]
+pub struct KeyLevelUpdate {
+    pub issuer: Issuer,
+    pub base_version: u64,
+    pub kid: KID,
+    pub to_upsert: Option<JWK>, // If none, it is a deletion.
+}
+
+impl KeyLevelUpdate {
+    pub fn as_issuer_level_repr(&self) -> anyhow::Result<ProviderJWKs> {
```

Comment:
nit: the naming is not very consistent, one has try_ prefix and another doesn't both have Result as return type